doc/releases/changelog-dev.md

```diff
@@ -74,6 +74,36 @@
   Array([0.25, 0.25, 0.25, 0.25], dtype=float64))
   ```
 
+* Catalyst now supports automatic qubit management.
+  [(#1788)](https://github.com/PennyLaneAI/catalyst/pull/1788)
+
+  The number of wires does not need to be speficied during device initialization,
+  and can be automatically allocated whenever a new wire is encountered.
```

Comment:
Let's try not to be too descriptive about _how_ it works, since the implementation could change. We can focus on what changes for the user.
```suggestion
  and instead will be automatically managed by the Catalyst Runtime.
```